prow/config.yaml

```diff
@@ -237,6 +237,45 @@ presubmits:
     rerun_command: "/test pull-kubernetes-cross"
     trigger: "(?m)^/test pull-kubernetes-cross,?(\\s+|$)"
     run_if_changed: '^(build\/|hack\/lib\/)|(Makefile)|(.*_(windows|linux|osx|unsupported)(_test)?\.go)$'
+  - name: pull-kubernetes-dind-build
+    agent: kubernetes
+    context: pull-kubernetes-dind-build
+    always_run: false
+    rerun_command: "/test pull-kubernetes-dind-build"
+    trigger: "(?m)^/test pull-kubernetes-dind-build,?(\\s+|$)"
+    spec:
+      containers:
+      - name: dind
+        image: docker:dind
+        securityContext:
+          priveleged: true
+        volumeMounts:
+        - name: docker-graph-storage
+          mountPath: /var/lib/docker
+      - name: build
+        image: gcr.io/k8s-testimages/e2e-prow
+        env:
+        - name: DOCKER_HOST
+          value: tcp://localhost:2375
```

Comment:
where does this come from? is this speaking to the actual docker daemon on the node?

Reply:
noting: email reply below, `dind` exposes docker on this port